packages/lodestar/test/unit/cli/commands/deposit.test.ts

```diff
@@ -1,3 +1,4 @@
+import {describe} from "mocha";
```

Comment:
probably not needed

Reply:
I'm adding this because we did something to tsconfig so both visual code and webstorm doesnt autoload mocha types. it works again if I add `types: [mocha]` to tsconfig but that key ovverides `typeRoots` so it doesn't work